### PR TITLE
feat: Remove favorited items & start using skip instead of pages

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR introduces the following:
- The capability to remove favorited items and make them disappear from the My Lists UI.
- The usage of skip instead of pages, which allows the removal of elements to not break the pagination.
- Simplifies the `AssetList` component so it receives assets instead of nfts or items.

### CODE REVIEW

File: webapp/src/components/InfiniteScroll/InfiniteScroll.tsx [hunk 1]
```diff
@@ -24,30 +20,27 @@ export function InfiniteScroll({
...
-    setScrollPage(0)
-  }, [onLoadMore, page])
+    onLoadMore()
+    setScrolledTimes(0)
+  }, [onLoadMore])
+
+  const showLoadMoreButton =
+    !isLoading &&
+    maxInfiniteScrolls !== undefined &&
+    scrolledTimes === maxInfiniteScrolls
```
Comment (kevinszuchet) [inline webapp/src/components/InfiniteScroll/InfiniteScroll.tsx:43]: `scrolledTimes` cannot be greater than `maxInfiniteScrolls` right?

File: webapp/src/modules/favorites/sagas.spec.ts [hunk 2]
```diff
@@ -29,11 +26,7 @@ import {
 } from './actions'
 import { favoritesSaga } from './sagas'
 import { getListId } from './selectors'
-import {
-  FETCH_ITEMS_SUCCESS,
-  fetchItemsRequest,
-  fetchItemsSuccess
-} from '../item/actions'
+import { fetchItemsRequest, fetchItemsSuccess } from '../item/actions'
```
Comment (kevinszuchet) [inline webapp/src/modules/favorites/sagas.spec.ts:29]: Would you mind moving this up 🙏 ?

**Review by kevinszuchet — COMMENTED** [review]
Amazing job 🚀 

I left 2 small comments

File: webapp/src/components/InfiniteScroll/InfiniteScroll.tsx (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/components/InfiniteScroll/InfiniteScroll.tsx:43]: That's right! Whenever it reaches scrolled times, the `load more button` appears and upon clicking it, it resets the counter.

File: webapp/src/modules/routing/sagas.spec.ts [hunk 3]
```diff
@@ -245,7 +243,7 @@ describe('when handling the browse action', () => {
     browseOptions = {
       onlyOnSale: undefined,
       sortBy: undefined,
-      page: 1,
+      skip: 1,
```
Comment (meelrossi) [inline webapp/src/modules/routing/sagas.spec.ts:246]: shouldn't this be 0? or 1 is ok?

**Review by meelrossi — DISMISSED** [review]

**Review by cyaiox — APPROVED** [review]
🚀

File: webapp/src/modules/ui/browse/selectors.ts [hunk 4]
```diff
@@ -23,8 +23,10 @@ import { ItemState } from '../../item/reducer'
 import { VendorName } from '../../vendor'
 import { getAddress, getWallet } from '../../wallet/selectors'
 import { getTransactionsByType } from '../../transaction/selectors'
+import { getFavoritedItems as getFavoritedItemsFromState } from '../../favorites/selectors'
 import { View } from '../types'
 import { OnRentNFT, OnSaleElement, OnSaleNFT } from './types'
+import { FavoritesData } from '../../favorites/types'
```
Comment (juanmahidalgo) [inline webapp/src/modules/ui/browse/selectors.ts:29]: can we move this import a few lines above? 🙏

**Review by juanmahidalgo — COMMENTED** [review]
More of a general question, if we're using `skip` now, should we also use `first`? 
In the past, `page` 2 (having a page size of 10 items) would be the combination of `first: 10`, `skip: 10`. Am I missing something or the `skip` is enough to replicate the old behavior?

**Comment by LautaroPetaccio** [comment]
Closed due to #1614

**Pull request closed without merging**